platform/pulpcore/app/serializers/repository.py

```diff
@@ -103,42 +105,41 @@ class ImporterSerializer(MasterModelSerializer, NestedHyperlinkedModelSerializer
         required=False,
     )
     ssl_validation = serializers.BooleanField(
-        help_text='Indicates whether SSL peer validation must be performed.',
+        help_text='If True, SSL peer validation must be performed.',
         required=False,
     )
     proxy_url = serializers.CharField(
-        help_text='The optional proxy URL. Format: scheme://user:password@host:port',
+        help_text='The proxy URL. Format: scheme://user:password@host:port',
         required=False,
     )
-    basic_auth_user = serializers.CharField(
-        help_text='The username to be used in HTTP basic authentication when syncing.',
+    username = serializers.CharField(
+        help_text='The username to be used for authentication when syncing.',
         write_only=True,
         required=False,
     )
-    basic_auth_password = serializers.CharField(
-        help_text='The password to be used in HTTP basic authentication when syncing.',
+    password = serializers.CharField(
+        help_text='The password to be used for authentication when syncing.',
         write_only=True,
         required=False,
     )
-    download_policy = serializers.ChoiceField(
-        help_text='The policy for downloading content.',
-        allow_blank=False,
-        choices=models.Importer.DOWNLOAD_POLICIES,
-    )
     last_sync = serializers.DateTimeField(
         help_text='Timestamp of the most recent successful sync.',
         read_only=True
     )
+    last_updated = serializers.DateTimeField(
+        help_text='Timestamp of the most recent update of the importer configuration.',
+        read_only=True
+    )
 
     repository = RepositoryRelatedField()
 
     class Meta:
         abstract = True
         model = models.Importer
         fields = MasterModelSerializer.Meta.fields + (
-            'name', 'last_updated', 'feed_url', 'validate', 'ssl_ca_certificate',
+            'name', 'feed_url', 'download_policy', 'sync_mode', 'validate', 'ssl_ca_certificate',
             'ssl_client_certificate', 'ssl_client_key', 'ssl_validation', 'proxy_url',
-            'basic_auth_user', 'basic_auth_password', 'download_policy', 'last_sync', 'repository',
+            'username', 'password', 'last_sync', 'last_updated', 'repository',
```

Comment:
Large paragraph style lists are, IMHO, less readable.  consider:
```
...Meta.fields + (
    'name',
    'feed_url',
    ...
)
```

Also, easier to keep alphabetized (if so inclined).

Sorry, in the mood to be pedantic.